src/javascripts/helpers/data/authData.js

```diff
@@ -11,10 +12,12 @@ const checkLoginStatus = () => {
       authDiv.addClass('hide');
       homeDiv.removeClass('hide');
       logoutButton.removeClass('hide');
+      $('#plane-collection').show(planesList.showPlanes);
```

Comment:
[`$().show()`](https://api.jquery.com/show/) doesn't take any args...I think what you want to do is `planesList.showPlanes()`.

also since you're showing the planes regardless of auth state, you can do it outside of the if/else. Probably outside of `authData` alltogether